### PR TITLE
Switchtools fix

### PR DESCRIPTION
**Pull request by Intigracy**
Fix https://github.com/vgstation-coders/vgstation13/issues/15182

This was created using the web editor on a shoddy mobile connection, lemme know if anything is funky or this doesn't work.

### CODE REVIEW

**Comment by DrSnips** [comment]
I can give it a test.

**Comment by Intigracy** [comment]
Does it work

File: code/game/objects/items/weapons/switchtool.dm [hunk 1]
```diff
@@ -260,26 +256,26 @@
...
+			return 1
 	//Joke module about power[clean/creep], this is dumb but exists.
 	//How does a UV light clean even? It just sterilizes. I guess it works because it's like suit storages with their UV suit cleaner.
 		if(istype(disk_tech, /datum/tech/powerstorage) && disk_tech.level >= 4)
 			stored_modules["/obj/item/weapon/soap/holo:UV sterilizer"] = new /obj/item/weapon/soap/holo(src)
 			to_chat(user, "The holo switchtool has a power clean design now!")
-			return
-
+			return 1
+
```
Comment (N3X15) [inline code/game/objects/items/weapons/switchtool.dm]: WHITESPACE AT END OF LINE REEEEEEEEEEEEEEEEE
Other than that, it looks ok

File: code/game/objects/items/weapons/switchtool.dm [hunk 2]
```diff
@@ -61,9 +61,6 @@
 		choose_deploy(user)
 
 /obj/item/weapon/switchtool/attackby(var/obj/item/used_item, mob/user)
-	if(istype(used_item, removing_item) && /obj/item/weapon/switchtool/holo)
```
Comment (Exxion) [inline code/game/objects/items/weapons/switchtool.dm:64]: I see what the error here was, but honestly I have a hard time blaming anyone except BYOND for this
(I mean other than the fact that this was how I thought it should work from the beginning, but disagreement is fine)

**Comment by DrSnips** [comment]
Sorry I died the last few days, but if you are going to play with the switchtool code, might as well fix the issue resolved from this pr #15130

**Comment by Intigracy** [comment]
What was the comment

**Comment by Intigracy** [comment]
Also I'll redo this on a local branch so it's not a mess, and I'll actually test it.

**Pull request closed without merging**